<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="5" passed="0" failed="1" skipped="4">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-05-08T00:20:23Z" name="Suite A" finished-at="2019-05-08T00:20:24Z" duration-ms="1118">
    <groups>
    </groups>
    <test started-at="2019-05-08T00:20:23Z" name="Login Test" finished-at="2019-05-08T00:20:24Z" duration-ms="896">
      <class name="HybridFramework.suites.LoginTest">
        <test-method is-config="true" signature="init()[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:23Z" name="init" finished-at="2019-05-08T01:20:24Z" duration-ms="476" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method is-config="true" signature="initTest()[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="initTest" finished-at="2019-05-08T01:20:24Z" duration-ms="101" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method signature="loginTest(java.util.Hashtable)[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="loginTest" data-provider="getData" finished-at="2019-05-08T01:20:24Z" duration-ms="3" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=example@mail.com, ExpectedResult=LoginSuccess, Password=Pass123, Runmode=N, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to: N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to: N
	at HybridFramework.suites.LoginTest.loginTest(LoginTest.java:24)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="quit()[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="quit" finished-at="2019-05-08T01:20:24Z" duration-ms="214" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method is-config="true" signature="initTest()[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="initTest" finished-at="2019-05-08T01:20:24Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method signature="loginTest(java.util.Hashtable)[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="loginTest" data-provider="getData" finished-at="2019-05-08T01:20:24Z" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=example@mail.com, ExpectedResult=LoginFailure, Password=Pass123, Runmode=N, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to: N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to: N
	at HybridFramework.suites.LoginTest.loginTest(LoginTest.java:24)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="quit()[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="quit" finished-at="2019-05-08T01:20:24Z" duration-ms="43" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method signature="loginTest(java.util.Hashtable)[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="loginTest" data-provider="getData" finished-at="2019-05-08T01:20:24Z" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=U3, ExpectedResult=LoginFailure, Password=P3, Runmode=N, Browser=Edge}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to: N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to: N
	at HybridFramework.suites.LoginTest.loginTest(LoginTest.java:24)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="initTest()[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="initTest" finished-at="2019-05-08T01:20:24Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method is-config="true" signature="quit()[pri:0, instance:HybridFramework.suites.LoginTest@30ee2816]" started-at="2019-05-08T01:20:24Z" name="quit" finished-at="2019-05-08T01:20:24Z" duration-ms="41" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- HybridFramework.suites.LoginTest -->
    </test> <!-- Login Test -->
    <test started-at="2019-05-08T00:20:24Z" name="TestB" finished-at="2019-05-08T00:20:24Z" duration-ms="131">
      <class name="HybridFramework.suites.TestB">
        <test-method is-config="true" signature="init()[pri:0, instance:HybridFramework.suites.TestB@393671df]" started-at="2019-05-08T01:20:24Z" name="init" finished-at="2019-05-08T01:20:24Z" duration-ms="40" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method is-config="true" signature="initTest()[pri:0, instance:HybridFramework.suites.TestB@393671df]" started-at="2019-05-08T01:20:24Z" name="initTest" finished-at="2019-05-08T01:20:24Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method signature="testB(java.util.Hashtable)[pri:0, instance:HybridFramework.suites.TestB@393671df]" started-at="2019-05-08T01:20:24Z" name="testB" data-provider="getData" finished-at="2019-05-08T01:20:24Z" duration-ms="55" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=fdsfsd, ExpectedResult=fsd, Password=fsdfsd, Runmode=Y, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.reflect.InvocationTargetException">
            <full-stacktrace>
              <![CDATA[java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at HybridFramework.drivers.DriverScript.executeKeywords(DriverScript.java:54)
	at HybridFramework.suites.TestB.testB(TestB.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
Caused by: java.lang.NoClassDefFoundError: kotlin/TypeCastException
	at okhttp3.ResponseBody.create(ResponseBody.java:209)
	at okhttp3.internal.Util.<clinit>(Util.java:51)
	at okhttp3.ConnectionPool.<clinit>(ConnectionPool.java:52)
	at org.openqa.selenium.remote.internal.OkHttpClient$Factory.<init>(OkHttpClient.java:118)
	at org.openqa.selenium.remote.internal.OkHttpClient$Factory.<init>(OkHttpClient.java:123)
	at org.openqa.selenium.remote.http.HttpClient$Factory.createDefault(HttpClient.java:59)
	at org.openqa.selenium.remote.HttpCommandExecutor.<clinit>(HttpCommandExecutor.java:47)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:181)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:168)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:123)
	at HybridFramework.keywords.GenericKeywords.openBrowser(GenericKeywords.java:60)
	... 30 more
Caused by: java.lang.ClassNotFoundException: kotlin.TypeCastException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 41 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.reflect.InvocationTargetException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testB -->
        <test-method is-config="true" signature="quit()[pri:0, instance:HybridFramework.suites.TestB@393671df]" started-at="2019-05-08T01:20:24Z" name="quit" finished-at="2019-05-08T01:20:24Z" duration-ms="34" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- HybridFramework.suites.TestB -->
    </test> <!-- TestB -->
    <test started-at="2019-05-08T00:20:24Z" name="Jobserve" finished-at="2019-05-08T00:20:24Z" duration-ms="68">
      <class name="HybridFramework.suites.Jobserve">
        <test-method is-config="true" signature="init()[pri:0, instance:HybridFramework.suites.Jobserve@6973b51b]" started-at="2019-05-08T01:20:24Z" name="init" finished-at="2019-05-08T01:20:24Z" duration-ms="35" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method is-config="true" signature="initTest()[pri:0, instance:HybridFramework.suites.Jobserve@6973b51b]" started-at="2019-05-08T01:20:24Z" name="initTest" finished-at="2019-05-08T01:20:24Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTest -->
        <test-method is-config="true" signature="quit()[pri:0, instance:HybridFramework.suites.Jobserve@6973b51b]" started-at="2019-05-08T01:20:24Z" name="quit" finished-at="2019-05-08T01:20:24Z" duration-ms="30" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method signature="JobserveTest(java.util.Hashtable)[pri:0, instance:HybridFramework.suites.Jobserve@6973b51b]" started-at="2019-05-08T01:20:24Z" name="JobserveTest" data-provider="getData" finished-at="2019-05-08T01:20:24Z" duration-ms="0" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=example@mail.com, Runmode=N, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode is set to: N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode is set to: N
	at HybridFramework.suites.Jobserve.JobserveTest(Jobserve.java:22)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1181)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- JobserveTest -->
      </class> <!-- HybridFramework.suites.Jobserve -->
    </test> <!-- Jobserve -->
  </suite> <!-- Suite A -->
  <suite started-at="2019-05-08T00:20:24Z" name="My suite" finished-at="2019-05-08T00:20:24Z" duration-ms="0">
    <groups>
    </groups>
  </suite> <!-- My suite -->
</testng-results>
